storage/simulation/main.go

```diff
@@ -27,18 +28,21 @@ func main() {
 	stopper := stop.NewStopper()
 	defer stopper.Stop()
 
-	c := createCluster(stopper, 5)
+	// TODO(bram): Setup flags to allow filenames for these outputs. Print both
+	// to action and epoch to os.Stdout as well.
+	epochWriter := os.Stdout
```

Comment:
`epochWriter := tabwriter.NewWriter(os.Stdout, 2, 1, 2, ' ', 0)`

The constants above are taken from elsewhere in the codebase; tweak as needed.

Ditto on the next line.